modules/Sources/Features/BalanceBreakdown/BalanceBreakdownView.swift

```diff
@@ -77,8 +77,10 @@ extension BalanceBreakdownView {
                     tokenName
                 )
             )
-            .font(.system(size: 32))
-            .fontWeight(.bold)
+            .font(
+                .custom(FontFamily.Inter.regular.name, size: 32)
```

Comment:
This is out of the scope of this task, but it's still worth asking. I see a pattern of adding this extra font parameter to many texts on different screens. Could we extra commonly set parameters as default parameters to a new UI component which we then use on several screens?

Reply:
And second related question, why some of the UI text components here do not use the font parameter like the one on line 72?

Reply:
> This is out of the scope of this task, but it's still worth asking. I see a pattern of adding this extra font parameter to many texts on different screens. Could we extra commonly set parameters as default parameters to a new UI component which we then use on several screens?

There's not much I can do except making a view modifier that does almost the same, except the font. For example:
```
.interFont(32)
```

Maybe nice to have, I can create a followup ticket.

Reply:
> And second related question, why some of the UI text components here do not use the font parameter like the one on line 72?

The font is propagated from the parent. I set default Inter font to the root of the app so all Text instances use Inter in a default setup. Whenever needed, this must be overloaded by local .font instance.